### ISSUE TITLE
--addUTR and --useexisting

### ISSUE THREAD

**Issue by krabapple** [Gaius-Augustus/BRAKER#369]
In the BRAKER manual, the recommended script for  an --addUTR run to generate UTRs for a previous run is

```
braker.pl --genome=../genome.fa --addUTR=on --softmasking \
    --bam=../RNAseq.bam --workingdir=$wd \
    --AUGUSTUS_hints_preds=augustus.hints.gtf --cores=8 \
    --skipAllTraining --species=somespecies
```

It is recommended that 'somespecies' be the same species folder as was used in the original run.

But I notice --useexisting is not mentioned....is it superfluous (or even discouraged) in this use case?

**Comment by tomasbruna** [comment]
That's correct, when the `--skipAllTraining` option is used, `--useexisting` is automatically set to True. So it is superfluous in this case.
